docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -504,17 +504,7 @@ Default: `false`
 
 ## Tablet splitting flags
 
-##### --enable_automatic_tablet_splitting
```

Comment:
@hari90 , Many customers have been dabbling around with tablet splitting gflag for a couple of reasons. Please leave this gflag as is. Otherwise, it can result in more questions on slack/CEs on how to enable / disable.